sdk/translation/ai-translation-text/README.md

```diff
@@ -0,0 +1,317 @@
+# Azure TextTranslation REST client library for JavaScript
+
+Text translation is a cloud-based REST API feature of the Translator service that uses neural
+machine translation technology to enable quick and accurate source-to-target text translation
+in real time across all supported languages.
+
+The following methods are supported by the Text Translation feature:
+
+Languages. Returns a list of languages supported by Translate, Transliterate, and Dictionary Lookup operations.
+
+Translate. Renders single source-language text to multiple target-language texts with a single request.
+
+Transliterate. Converts characters or letters of a source language to the corresponding characters or letters of a target language.
+
+Detect. Returns the source code language code and a boolean variable denoting whether the detected language is supported for text translation and transliteration.
+
+Dictionary lookup. Returns equivalent words for the source term in the target language.
+
+Dictionary example Returns grammatical structure and context examples for the source term and target term pair.
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+Key links:
+
+- [Package (NPM)](https://www.npmjs.com/package/@azure-rest/ai-translation-text)
+- [API reference documentation](https://learn.microsoft.com/azure/cognitive-services/translator/reference/v3-0-reference)
+- [Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/translation/ai-translation-text/samples)
+
+## Getting started
+
+### Currently supported environments
+
+- LTS versions of Node.js
```

Comment:
Can this package work in the browser?

Reply:
I believe it should technically work in browsers. I will check our partner SDK - document translator

Reply:
Maybe Node is the only environment for now: https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/documenttranslator/ai-document-translator-rest#currently-supported-environments

Reply:
If your tests work in the browser, then I don't see a strong reason why not to support browsers. It could be the case that ai-document-translator's readme accidentally didn't include the browser support. See this [section](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/template/template#browser-support) for more details.

Reply:
Added browser support to the Readme. As there are no configuration of CORS in Cognitive Services, I have not added that section.